src/data/graphql/typeDefs.ts

```diff
@@ -19,10 +19,19 @@ export default gql`
   }
 
   type ParsedEvent {
+    type: String!
     name: String!
     values: String!
     createdAt: Int!
     emmitedBy: String!
+    blockNumber: Int
```

Comment:
Actually I don't think so. Good catch

Reply:
Actually, this is correct. We are sharing this type between a few resolver returns and we don't return the `blockNumber` for all of them.

So yeah, it might be `null` in some cases.